galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager/src/main/java/dev/galasa/cicsts/cemt/CEMTManagerField.java

```diff
@@ -10,8 +10,12 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import dev.galasa.framework.spi.ValidAnnotatedFields;
+
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE })
+@CEMTManagerField
```

Comment:
um, no.   you dont put the managerfield annotation on the managerfield annotation